src/rest-api/trainer/trainerApi.ts

```diff
@@ -14,6 +16,19 @@ class TrainerApi {
 
     return Promise.resolve(content);
   }
+  public getExerciseEvaluationById(id: number): Promise<ExerciseEvaluation> {
+    let evaluation: ExerciseEvaluation;
+
+    const evaluationData = exerciseEvaluationMockedData.filter((evaluationItem) => {
```

Comment:
Instead of use `filter` you can use [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) to get the first item that satisfies the condition